crates/forge_domain/src/tools.rs

```diff
@@ -661,15 +661,21 @@ impl Tools {
     pub fn contains(tool_name: &ToolName) -> bool {
         FORGE_TOOLS.contains(tool_name)
     }
-    pub fn is_complete(tool_name: &ToolName) -> bool {
+    pub fn should_yield(tool_name: &ToolName) -> bool {
         // Tools that convey that the execution should yield
         [
-            ToolsDiscriminants::ForgeToolFollowup,
             ToolsDiscriminants::ForgeToolAttemptCompletion,
+            ToolsDiscriminants::ForgeToolFollowup,
         ]
         .iter()
         .any(|v| v.to_string().to_case(Case::Snake).eq(tool_name.as_str()))
     }
+    pub fn is_complete(tool_name: &ToolName) -> bool {
+        // Tool that convey that conversation might be completed
+        [ToolsDiscriminants::ForgeToolAttemptCompletion]
+            .iter()
+            .any(|v| v.to_string().to_case(Case::Snake).eq(tool_name.as_str()))
+    }
```

Comment:
Lets revert changes here.

Reply:
What i want to do is only on AttemptCompletion send the ChatResponse::ChatComplete message to display the summary, but follow up should also end the turn (but without showing the summary). i will try to use better naming for it